### ISSUE TITLE
hostapd socket timeout occaasionally.

### ISSUE THREAD

**Issue by Bairdo** [Bairdo/gasket#41]
possibly when faucet has not started up yet

```
hub: uncaught exception: Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/ryu/lib/hub.py", line 60, in _launch
    return func(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/gasket/auth_app.py", line 158, in run
    self._init_unsolicited_socket()
  File "/usr/lib/python3.6/site-packages/gasket/auth_app.py", line 90, in _init_unsolicited_socket
    self.logger)
  File "/usr/lib/python3.6/site-packages/gasket/hostapd_ctrl.py", line 286, in unsolicited_socket_udp
    s = request_socket_udp(host, port, bind_address, bind_port, hsoc_type, timeout, logger)
  File "/usr/lib/python3.6/site-packages/gasket/hostapd_ctrl.py", line 273, in request_socket_udp
    return HostapdCtrlUDP(addrinfo[0], addrinfo[4], bind_address, bind_port, hsoc_type, timeout, logger)
  File "/usr/lib/python3.6/site-packages/gasket/hostapd_ctrl.py", line 232, in __init__
    self.connect(sockaddr)
  File "/usr/lib/python3.6/site-packages/gasket/hostapd_ctrl.py", line 243, in connect
    c = self.get_cookie()
  File "/usr/lib/python3.6/site-packages/gasket/hostapd_ctrl.py", line 258, in get_cookie
    return str(self.request('GET_COOKIE'))
  File "/usr/lib/python3.6/site-packages/gasket/hostapd_ctrl.py", line 38, in request
    return self.receive(size=4096)
  File "/usr/lib/python3.6/site-packages/gasket/hostapd_ctrl.py", line 156, in receive
    return str(self.soc.recv(size))
  File "/usr/lib/python3.6/site-packages/eventlet/greenio/base.py", line 354, in recv
    return self._recv_loop(self.fd.recv, b'', bufsize, flags)
  File "/usr/lib/python3.6/site-packages/eventlet/greenio/base.py", line 348, in _recv_loop
    self._read_trampoline()
  File "/usr/lib/python3.6/site-packages/eventlet/greenio/base.py", line 319, in _read_trampoline
    timeout_exc=socket.timeout("timed out"))
  File "/usr/lib/python3.6/site-packages/eventlet/greenio/base.py", line 203, in _trampoline
    mark_as_closed=self._mark_as_closed)
  File "/usr/lib/python3.6/site-packages/eventlet/hubs/__init__.py", line 162, in trampoline
    return hub.switch()
  File "/usr/lib/python3.6/site-packages/eventlet/hubs/hub.py", line 294, in switch
    return self.greenlet.switch()
socket.timeout: timed out
```

**Comment by Bairdo** [comment]
happening more than 'occasionally'

**Comment by Bairdo** [comment]
this particular issue fixed with #47 which removes no longer uses eventlet.
